src/Installers/Windows/AspNetCoreModule-Setup/ANCMV2/AncmV2.wixproj

```diff
@@ -7,7 +7,7 @@
     <IsShipping>true</IsShipping>
     <ProjectGuid>f9bacb48-3bd7-4ec2-ae31-664e8703ec12</ProjectGuid>
     <OutputType>Package</OutputType>
-    <OutputName>aspnetcoremodule_$(Platform)_en_v2</OutputName>
+    <OutputName>aspnetcoremodule_$(Platform)_en_v2_$(ANCMFolderVersion)</OutputName>
```

Comment:
Does 'ANCMFolderVersion' change on the same cadence as 'PackageVersion'?

Reply:
It changes as frequently as the MSI version does (which is every day). That should be fine for our needs.